dnf.spec

```diff
@@ -65,6 +66,7 @@ Provides:       dnf-command(search)
 Provides:       dnf-command(updateinfo)
 Provides:       dnf-command(upgrade)
 Provides:       dnf-command(upgrade-to)
+Conflicts:      dnf-plugins-core < %{min_plugins_core}
```

Comment:
Yes, I see there is no backward requirement in python2/python3. I am just wondering why it was done this way.